src/components/pages/situation-page/SituationPage.tsx

```diff
@@ -2,15 +2,12 @@ import React from 'react';
 import { ComponentMapper } from '../../ComponentMapper';
 import { SituationPageProps } from '../../../types/content-props/dynamic-page-props';
 import { ThemedPageHeader } from '../../_common/headers/themed-page-header/ThemedPageHeader';
-import { BEM } from '../../../utils/classnames';
-
-const bem = BEM('situation-page');
 
 export const SituationPage = (props: SituationPageProps) => {
     return (
-        <div className={bem()}>
+        <div className={'situationPage'}>
             <ThemedPageHeader contentProps={props} />
-            <div className={bem('content')}>
+            <div className={'content'}>
```

Comment:
trenger vi ikke fortsatt BEM her for å skille fra content i ProductPage.scss?

Reply:
Nei, fordi den ligger hierarikisk under .situationPage